demonstrations/tutorial_qnn_multivariate_regression.py

```diff
@@ -0,0 +1,289 @@
+
+wires=2
+layers=4
+params_shape = qml.StronglyEntanglingLayers.shape(n_layers=layers,n_wires=wires)
+params=np.random.default_rng().random(size=params_shape)
+best_params=optimization_jit(params, x_train, jnp.array(y_train), print_training=True)
+
+######################################################################
+# Once the optimized :math:`\vec{\theta}` has been obtained, we can use those parameters to build our fitted version of the function.
+
+def evaluate(params, data):
+    y_pred = jax.vmap(quantum_neural_network, in_axes=(None, 0))(params, data)
+    return y_pred
+y_predictions=evaluate(best_params,x_train)
+
+######################################################################
+# To compare the fitted function to the exact target function, let's take a look at the :math:`R^2` score:
+
+from sklearn.metrics import r2_score
+r2 = round(float(r2_score(y_train, y_predictions)),3)
+print("R^2 Score:", r2)
+
+######################################################################
+# Let's now plot the results to visually check how good our fit is!
+
+fig = plt.figure()
+# Target function
+ax1 = fig.add_subplot(1, 2, 1, projection='3d')
+ax1.plot_surface(x1_mesh,x2_mesh, y_train.reshape(x1_mesh.shape), cmap='viridis')
+ax1.set_zlim(0,1)
+ax1.set_xlabel('$x$',fontsize=10)
+ax1.set_ylabel('$y$',fontsize=10)
+ax1.set_zlabel('$f(x,y)$',fontsize=10)
+ax1.set_title('Target ')
+
+# Predictions
+ax2 = fig.add_subplot(1, 2, 2, projection='3d')
+ax2.plot_surface(x1_mesh,x2_mesh, y_predictions.reshape(x1_mesh.shape), cmap='viridis')
+ax2.set_zlim(0,1)
+ax2.set_xlabel('$x$',fontsize=10)
+ax2.set_ylabel('$y$',fontsize=10)
+ax2.set_zlabel('$f(x,y)$',fontsize=10)
+ax2.set_title(f' Predicted \nAccuracy: {round(r2*100,3)}%')
+
+# Show the plot
+plt.tick_params(axis='both', which='major', labelsize=8)
+plt.tight_layout(pad=4.0)
+
+######################################################################
+# Conclusions
+# ------------------------------------------
+# In this demo we have shown how to leverage the Fourier series representation 
+# of a variational quantum circuit to solve a regression problem for a two dimensional function. 
+# In particular we used JAX, an auto differentiable machine learning framework to accelerate the classical
+# optimization of the parameters. The results show a good agreement with the target function and the model 
+# can be trained further, increasing number of iterations in the training to maximize the accuracy. It also 
+# paves the way for addressing a regression problem for a :math:`N`-dimensional function, as everything presented 
+# here can be easily generalized. A final check that could be done is to obtain the Fourier coefficients of the
+# trained circuit and compare it with the Fourier series we obtained when directly from the circuit.
+
+######################################################################
```

Comment:
Will deleting this line of # fix it? I don't see it in other demos

Reply:
Nop, the output it's still the same :(